contracts/connectors/loantoken/LoanTokenLogicStandard.sol

```diff
@@ -1004,6 +1004,24 @@ contract LoanTokenLogicStandard is AdvancedToken, StorageExtension {
 
     /* Internal View functions */
 
+    function _nextBorrowInterestRate(
```

Comment:
there is original `function _nextBorrowInterestRate(` if its not needed we remove. if its needed we call new one from there

Reply:
the original one is needed as called by protocol contract. This one is internal and only used for proper estimations.

Reply:
looks good now